### DISCUSSION TITLE
Question about 2024 Robot Wiring

### DISCUSSION THREAD

**Discussion by honzikschenk** [team401/2024-Robot-Code#66] [General]
@PChild Why are we using krakens as the index motors? Why don't we use them as the swerve steer motors and/or put them on aiming? Do the index belts require more power? This isn't an issue in code, I'm just wondering.
Thanks

**Comment by PChild** [comment]
Krakens aren't on aiming because there's no MaxPlanetary adapter for the Kraken output shaft yet. We can use Falcons there because the Thrifty 8mm shaft lets us. 

Steering vs indexing motor allocation comes down to inventory that we have on hand - with motors arranged per the sheet we end up with 2 spare Krakens and 9 spare Falcons. If we swap to Kraken steer and Falcon indexing we would have zero spare Krakens. That seems bad. I don't think there's any real performance difference in Falcon vs Kraken steering so imo this is a better use of our available resources.

Also if we did run into a stall situation in the indexing system I do think the additional torque of the Krakens might be useful.

**Reply by honzikschenk** [reply]
Alright, that makes sense. Thank you
